Library/Sender.ts

```diff
@@ -15,8 +15,8 @@ import Util = require("./Util");
 import { URL } from "url";
 import Logging = require("./Logging");
 import { FileAccessControl } from "./FileAccessControl";
-import { Envelope } from "../Declarations/Contracts";
 
+const throttleStatusCode = 439; //  - Too many requests and refresh cache
```

Comment:
439 is Quota exceeded NOT too many requests.  
429 would be too many requests hence throttled.  Please see https://msazure.visualstudio.com/One/_git/Mgmt-AppInsights-Breeze?path=/Specifications/EndpointRequestResponse.md&_a=preview&version=GBmaster

Also, noticed that 439 is a legacy code.  They want new SDKs to use 402